v4/source/status.yaml

```diff
@@ -16,3 +21,44 @@
           $ref: '#/responses/BadRequest'
         '401':
           $ref: '#/responses/Unauthorized'
+    put:
+      tags:
+        - status
+      summary: Update user status
+      description: |
+        Update user status.
+        ##### Permissions
+        Must have `edit_other_users` permission for the team.
+      parameters:
+        - name: user_id
+          in: path
+          description: User ID
+          type: string
+        - name: body
+          in: body
+          description: Status object that is to be updated
+          required: true
+          schema:
+            type: object
+            required:
+              - status
+            properties:
+              user_id:
+                type: string
+              status:
+                type: string
+              manual:
+                type: boolean
+              last_activity_at:
+                type: integer
+              active_channel:
+                type: string
```

Comment:
For the properties can we only include `status`? The endpoint only uses that one so we should make sure that the documentation doesn't imply you can update the other fields